dev/com.ibm.ws.microprofile.openapi/src/com/ibm/ws/microprofile/openapi/impl/OpenAPIUIBundlesUpdater.java

```diff
@@ -32,7 +32,6 @@
 import org.osgi.framework.BundleContext;
 import org.osgi.framework.BundleException;
 import org.osgi.framework.FrameworkUtil;
-import org.osgi.framework.ServiceReference;
```

Comment:
The header here looks to still be using EPL-1.0.  It looks like you need to rebase your PR branch onto the latest `release` branch to be sure you are basing on the latest license headers.

Reply:
Also, update the copyright date once you rebase for the files you modified.